setup.py

```diff
@@ -1,67 +1,52 @@
-#!/usr/bin/env python
 # -*- coding: utf-8 -*-
-# -----------------------------------------------------------------------------
-# Copyright (c) Spyder Project Contributors
+# ----------------------------------------------------------------------------
+# Copyright © 2022, spyder-vim
 #
-# Licensed under the terms of the MIT License
-# (see LICENSE.txt for details)
-# -----------------------------------------------------------------------------
+# Licensed under the terms of the MIT license
+# ----------------------------------------------------------------------------
+"""
+spyder-vim setup.
+"""
+from setuptools import find_packages
+from setuptools import setup
 
-"""Setup script for spyder_vim"""
+from spyder_vim import __version__
 
-# Standard library imports
-import ast
-import os
-
-# Third party imports
-from setuptools import find_packages, setup
-
-HERE = os.path.abspath(os.path.dirname(__file__))
-
-
-def get_version(module='spyder_vim'):
-    """Get version."""
-    with open(os.path.join(HERE, module, '__init__.py'), 'r') as f:
-        data = f.read()
-    lines = data.split('\n')
-    for line in lines:
-        if line.startswith('VERSION_INFO'):
-            version_tuple = ast.literal_eval(line.split('=')[-1].strip())
-            version = '.'.join(map(str, version_tuple))
-            break
-    return version
-
-
-def get_description():
-    """Get long description."""
-    with open(os.path.join(HERE, 'README.rst'), 'r') as f:
-        data = f.read()
-    return data
-
-
-REQUIREMENTS = ['spyder>=3.2.0']
 
 setup(
-    name='spyder-vim',
-    version=get_version(),
-    packages=find_packages(exclude=['contrib', 'docs', 'tests*']),
-    keywords=['Spyder', 'Plugin', 'Vim'],
-    url='https://github.com/spyder-ide/spyder-vim',
-    license='MIT',
-    author='Joseph Martinot-Lagarde',
-    author_email='contrebasse@gmail.com',
-    description='A plugin to enable vim keybingins to the spyder editor',
-    long_description=get_description(),
-    install_requires=REQUIREMENTS,
-    include_package_data=True,
+    # See: https://setuptools.readthedocs.io/en/latest/setuptools.html
+    name="spyder-vim",
+    version=__version__,
+    author="spyder-vim",
+    author_email="contrebasse@gmail.com",
```

Comment:
Lets keep the original author name but also add something like `and the spyder-vim contributors` and also change the email to be the Spyder project email, so this could become something like:

```suggestion
    author="Joseph Martinot-Lagarde and the spyder-vim contributors",
    author_email="spyder.python@gmail.com",
```